packages/@storefront/flux-capacitor/test/unit/core/requests/utils.ts

```diff
@@ -572,12 +572,24 @@ suite('requests helpers', ({ expect, stub, spy }) => {
       )).to.eql({ a: 'b', c: 'd', e: 'f' });
     });
 
-    it('should merge source if tranformation returned falsey', () => {
+    it('should merge source if transformation returned falsey', () => {
       expect(RequestHelpers.chain(
         utils.normalizeToFunction(<any>{ a: 'b' }),
         (x) => null,
         utils.normalizeToFunction({ e: 'f' })
       )).to.eql({ a: 'b', e: 'f' });
     });
   });
+
+  describe('attachSessionId()', () => {
+    it('should return a function that is called with the given request and add sessionId to the request', () => {
+      const req = { a: 'b' };
+      const sessionId = 'foo';
+      const state: any = { c: 'd', session: { sessionId } };
+
+      const requestWithSessionId = RequestHelpers.attachSessionId(state)(req);
```

Comment:
Consider splitting this test into multiple assertions. Something that would:

- Invoke `attachSessionId`, capture the result, and assert that it's a function.
- Invoke the resulting function with the base request, capture the result, and assert that it contains the additional data.

Reply:
Done.